README.md

```diff
@@ -64,11 +64,16 @@ hive.allow-rename-table=true
 -   Want to report a bug or request a feature? Let us know on [Slack](http://slack.getdbt.com/), or open [an issue](https://github.com/fishtown-analytics/dbt-presto/issues/new).
 
 ### Running tests
+Build dbt container locally:
+
+```
+cd docker/
+./dbt/build.sh
```

Comment:
This actually didn't work for me due to something about paths. I had to run `./docker/dbt/build.sh` from the root. The script could be changed to match this instruction, or you can just change the instructions to not include `cd docker` and instead update the paths.

Reply:
Changed: https://github.com/fishtown-analytics/dbt-presto/pull/10/commits/47faed0f53dd5826f4de9c92141a148554a7e709